src/Core/src/Platform/Windows/NavigationRootManager.cs

```diff
@@ -89,33 +89,17 @@ public virtual void Connect(UIElement platformView)
 				_rootView.Content = null;
 			}
 
-			NavigationView rootNavigationView;
-			if (platformView is NavigationView nv)
+			_rootView.Content = platformView is NavigationView ? platformView : new RootNavigationView()
```

Comment:
This seems right :-) 

I think this existed before the if statement above it, so, at some point those if blogs were doing more